tests/common/test_case.py

```diff
@@ -95,8 +98,11 @@ def ensure_model_trains_and_loads(self, model_class, args: Params):
             # As above, we'll just pass on this.
             pass
         else:
-            assert_allclose(model.model.predict(model.validation_arrays[0]),
-                            loaded_model.model.predict(indexed_validation_arrays[0]))
+            model_predictions = model.model.predict(model.validation_arrays[0])
```

Comment:
Thanks for making this change - it wouldn't have been a bug we would have caught any time soon!